services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/PortHistorySpec.groovy

```diff
@@ -39,13 +42,14 @@ class PortHistorySpec extends HealthCheckSpecification {
         }
 
         then: "Port history is created on the src switch"
-        def timestampAfterDown = System.currentTimeMillis()
-        with(northboundV2.getPortHistory(isl.srcSwitch.dpId, isl.srcPort, timestampBefore, timestampAfterDown)) {
-            it.size() == 2
-            def antiFlapActivatedEvent = it.find { it.event == "ANTI_FLAP_ACTIVATED" }
-            checkPortHistory(antiFlapActivatedEvent, isl.srcSwitch.dpId, isl.srcPort, "ANTI_FLAP_ACTIVATED")
-            def portDownEvent = it.find { it.event == "PORT_DOWN" }
-            checkPortHistory(portDownEvent, isl.srcSwitch.dpId, isl.srcPort, "PORT_DOWN")
+        Wrappers.wait(WAIT_OFFSET) {
+            Long timestampAfterDown = System.currentTimeMillis()
+            with(northboundV2.getPortHistory(isl.srcSwitch.dpId, isl.srcPort, timestampBefore, timestampAfterDown)) {
+                it.size() == 2
+                checkPortHistory(it.find { it.event == "ANTI_FLAP_ACTIVATED" }, isl.srcSwitch.dpId, isl.srcPort,
```

Comment:
can we use `it[0]` instead of `.find { it.event == "ANTI_FLAP_ACTIVATED" }` here? I suppose the order of these two events is always the same